src/createContext.ts

```diff
@@ -147,10 +147,15 @@ export const importExternalSymbols = (context: Context, externalSymbols: string[
  */
 export const importBuiltins = (context: Context, externalBuiltIns: CustomBuiltIns) => {
   ensureGlobalEnvironmentExist(context)
-
+  const placeholder = Symbol()
   const rawDisplay = (v: Value, s: string) =>
     externalBuiltIns.rawDisplay(v, s, context.externalContext)
-  const display = (v: Value, s: string) => (rawDisplay(stringify(v), s), v)
+  const display = (v: Value, s: any = placeholder) => {
+    if (s !== placeholder && typeof s !== 'string') {
+      throw new Error('display expects the second argument to be a string')
```

Comment:
Is it possible to change `Error` to `TypeError`? That's the error thrown in https://github.com/source-academy/js-slang/blob/master/src/utils/rttc.ts